webpack/ForemanTasks/Routes/ForemanTasksRoutes.js

```diff
@@ -1,12 +1,17 @@
 import React from 'react';
-import IndexTasks from './IndexTasks';
+import TasksTableWrapper from '../Components/TasksTable';
 import ShowTask from './ShowTask';
 
 const ForemanTasksRoutes = {
   indexTasks: {
-    path: '/foreman_tasks/ex_tasks',
+    path: '/foreman_tasks/tasks',
     exact: true,
-    render: props => <IndexTasks {...props} />,
+    render: props => <TasksTableWrapper {...props} />,
+  },
+  subTasks: {
+    path: '/foreman_tasks/tasks/:id/sub_tasks',
+    exact: true,
+    render: props => <TasksTableWrapper isSubTask {...props} />,
```

Comment:
IMO `indexTasks` and `subTasks` should not use the `TasksTableWrapper` directly, instead the can have their own components. e.g. `<TasksIndexPage />` and `<SubTaskPage />`

Reply:
Why? They are the same and the only difference is the breadcrumbs

Reply:
At the future we might want to make 2 types of changes:
1. Changes that affect both of the pages. Now can be achieved easily
2. Changes that affect only one of the pages. Will be tricky to do ATM.

Another reason is to keep a unified convention for routes. ATM each route have its own component.

Reply:
If the subtasks component changes enough to be a different component then it should be a different component. But right now it has the same design and features as the index plus breadcrumbs. 
I don't think it would make sense to copy the whole TasksTableWrapper into 2 components which will have 2 of the same index files.
And if we just reuse TasksTableWrapper then changes that affect only one of the pages will be tricky to do anyway.

Reply:
I didn't mean removing the `TasksTableWrapper`, just creating 2 different components that uses the `TasksTableWrapper`. They should be very small and should just render the `TasksTableWrapper` with the right props.

I would consider renaming `TasksTableWrapper` to `TasksTablePage`.

Reply:
I agree on the name.
If both the components use `TasksTableWrapper` then changing one of them will still involve changing `TasksTableWrapper` for both